api/v1alpha1/types_etcd.go

```diff
@@ -449,3 +450,23 @@ func (e *Etcd) GetDeltaSnapshotLeaseName() string {
 func (e *Etcd) GetFullSnapshotLeaseName() string {
 	return fmt.Sprintf("%s-full-snap", e.Name)
 }
+
+// GetDefaultLabels returns the default labels for etcd.
+func (e *Etcd) GetDefaultLabels() map[string]string {
+	return map[string]string{
+		"name":     "etcd",
+		"instance": e.Name,
+	}
+}
+
+// GetEtcdAsOwnerReference returns an OwnerReference object that represents the current Etcd instance.
+func (e *Etcd) GetEtcdAsOwnerReference() metav1.OwnerReference {
```

Comment:
```suggestion
func (e *Etcd) GetAsOwnerReference() metav1.OwnerReference {
```
Not need to repeat `etcd` in the method name as it is the method on `etcd`. So when you call it will be `etcd.GetAsOwnerReference()` which clearly states what it does.